product_meat_unece/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Meat Unece",
+    "summary": """
+        This module adds the UNECE Meat Carcasses and Cuts Classification.""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://acsone.eu/",
```

Comment:
I think the website should be the one of the OCA repo

Reply:
Fixed